vendor/assets/javascripts/react-chosen.js

```diff
@@ -0,0 +1,69 @@
+(function(window, React, $) {
```

Comment:
Hmm. I just checked and they have the same versions. This would mean we can never pull any react components via rails-assets. So let's please try to figure this out. I don't really see why it would break if we pull the react.js from the rails-assets package...